src/components/query-list/query-list.js

```diff
@@ -84,9 +83,7 @@ export default class QueryList extends Component {
               {state.rejected[0].error[0].title}
             </div>
           ) : !state.length ? (
```

Comment:
In this case you can omit bracers
```suggestion
          ) : !state.length
            ? notFoundMessage
            : (
              <ScrollView
```